Dashboard/Views/ConnectionView.axaml.cs

```diff
@@ -35,7 +54,7 @@ private async void FileSelectionClicked(object sender, RoutedEventArgs args)
         }
     }
 
-    private void ConnectionTypeUpdated(object sender, RoutedEventArgs args)
+    private void ConnectionTypeUpdated(object? sender, RoutedEventArgs args)
     {
```

Comment:
I'm not sure that would be the best way of handling this, at least right now since the other methods of connections are not implemented, I think leaving it as is but adding the disconnect button and disabling the other connection types for now would be enough.